### ISSUE TITLE
Expected 4.0.1 release date?

### ISSUE THREAD

**Issue by ShaneCourtrille** [aspnet/AspNetKatana#216]
It looks like there are 5 issues linked to 4.0.1.  3 of them are fixed and the remaining 2 sound like easy fixes so it's a bit confusing why we haven't seen 4.0.1 already.  

I know major development efforts have moved over to ASP.Net Core but there are still some of us who have apps we cannot transition over so it would be nice to have a version of Microsoft.Owin we can use without the 5 known issues.

**Comment by Tratcher** [comment]
It's not scheduled at the moment. Is there a specific issue your blocked on?

**Comment by muratg** [comment]
@ShaneCourtrille Please let us know if there's any specific issue you'd like fixed in an upcoming patch. 

BTW, we still don't have a set date for a 4.0.1 release.
